corefoundation.gemspec

```diff
@@ -1,27 +1,27 @@
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path("lib", __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'corefoundation/version'
+require "corefoundation/version"
 
 Gem::Specification.new do |s|
-  s.name = %q{corefoundation}
+  s.name = "corefoundation"
   s.version = CF::VERSION
 
   s.authors = ["Frederick Cheung", "Chef Software, Inc."]
-  s.description = %q{Ruby wrapper for macOS Core Foundation framework}
-  s.summary = %q{Ruby wrapper for macOS Core Foundation framework}
+  s.description = "Ruby wrapper for macOS Core Foundation framework"
+  s.summary = "Ruby wrapper for macOS Core Foundation framework"
   s.email = ["frederick.cheung@gmail.com", "oss@chef.io"]
   s.files += Dir["lib/**/*.rb"]
   s.files += Dir["spec/**/*"]
-  s.files += ['README.md', 'LICENSE', 'CHANGELOG.md']
-  s.license = 'MIT'
-  s.homepage = %q{http://github.com/chef/corefoundation}
+  s.files += ["README.md", "LICENSE", "CHANGELOG.md"]
```

Comment:
Let's remove the readme and changelog here. We remove those from the chef install anyways to shave space.